packages/wct-mocha/README.md

```diff
@@ -0,0 +1,159 @@
+[![NPM version](http://img.shields.io/npm/v/wct-mocha?style=flat-square)](https://npmjs.org/package/wct-mocha)
+[![Build Status](http://img.shields.io/travis/Polymer/tools.svg?style=flat-square)](https://travis-ci.org/Polymer/tools)
+[![Gitter](http://img.shields.io/badge/slack-join%20chat%20%E2%86%92-brightgreen.svg?style=flat-square)](https://polymer-slack.herokuapp.com/)
+
+`wct-mocha` makes testing your web components a breeze!
+
+You get a streamlined browser-based testing environment, designed to work with [web-component-tester](https://github.com/Polymer/tools/tree/master/packages/web-component-tester) or on its own.
+
+
+# Getting Started
+
+## Install mocha and wct-mocha as devDependencies
+
+```bash
+$ npm install --save-dev mocha
+$ npm install --save-dev wct-mocha
+```
+
+## Install `web-component-tester` or `polymer-cli` globally
+
+```bash
+$ npm install --global web-component-tester
+```
+
+## Run `wct`
+
+```bash
+$ wct --wct-package-name wct-mocha
+```
+
+## `.html` Suites
+
+Your test suites can be `.html` documents. For example,
+`test/awesomest-tests.html`:
+
+```html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="../node_modules/mocha/mocha.js"></script>
+  <script src="../node_modules/chai/chai.js"></script>
+  <script src="../node_modules/wct-mocha/browser.js"></script>
```

Comment:
is it possible to pick a different name for browser.js now, or would that cause too much churn? maybe wct-mocha.js?

Reply:
Done: https://github.com/Polymer/tools/pull/738